radicle-cli/src/commands/push.rs

```diff
@@ -19,9 +19,10 @@ Usage
 
     By default, only the current branch is pushed.
 
+    --all               Push all branches (default: false)
+
 Options
 
-    --all               Push all branches (default: false)
```

Comment:
Keep here.

Reply:
Kept it here, and changed command from:
```
rad push [--all] [<option>...]
```
to:
```
rad push [<option>...]
```